.github/workflows/airflow-plugin.yml

```diff
@@ -57,14 +57,15 @@ jobs:
         if: always()
         run: source metadata-ingestion-modules/airflow-plugin/venv/bin/activate && pip freeze
       - uses: actions/upload-artifact@v3
+        if: ${{ always() && matrix.python-version == '3.10' && matrix.extraPythonRequirement == 'apache-airflow>2.6.0' }}
         with:
           name: Test Results (Airflow Plugin ${{ matrix.python-version }})
           path: |
             **/build/reports/tests/test/**
             **/build/test-results/test/**
             **/junit.*.xml
       - name: Upload coverage to Codecov
-        if: ${{ always() && matrix.python-version == '3.10' }}
+        if: ${{ always() && matrix.python-version == '3.10' && matrix.extraPythonRequirement == 'apache-airflow>2.6.0' }}
```

Comment:
it's actually fine to upload multiple coverage reports, as long as we set different flags on them. the main ingestion tests should be doing this already

Reply:
done